projects/CNSPRCY/programs/cnsprcy/module.nix

```diff
@@ -10,10 +10,11 @@ in
 {
   options.programs.cnsprcy = {
     enable = lib.mkEnableOption "cnsprcy";
-    package = lib.mkPackageOption pkgs "cnsprcy" { };
```

Comment:
If a program has a single executable, it's idiomatic to store this in a single option, because this will allow users to change and override it more easily.

Reply:
whoops, i accidentally deleted this line when porting CNSPRCY over to the project template. but i also didn't know what this options purpose was in the first place so thank you for the explanation @eljamm!